eng/testing/linker/SupportFiles/Directory.Build.targets

```diff
@@ -29,6 +36,25 @@
     </ItemGroup>
   </Target>
 
+  <!-- This target is just a workaround while https://github.com/dotnet/runtime/issues/48522 gets fixed. Once it is, this whole target
+  should be removed. -->
+  <Target Name="RootJavaScriptInteropAssembly"
+          BeforeTargets="PrepareForILLink"
+          Condition="'$(RuntimeIdentifier)' == 'browser-wasm'">
+    <ItemGroup>
+      <TrimmerRootAssembly Include="%(ManagedAssemblyToLink.Identity)" Condition="'%(ManagedAssemblyToLink.FileName)' == 'System.Private.Runtime.InteropServices.JavaScript'" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="CreateTestWasmAppBundle"
+          AfterTargets="Publish"
+          DependsOnTargets="BundleTestWasmApp"
+          Condition="'$(TargetArchitecture)' == 'wasm' And '$(TargetOs)' == 'browser'" />
+
+  <!-- Overriding this target as these projects won't need to binplace -->
+  <Target Name="GetBinPlaceTargetFramework" />
```

Comment:
test projects don't binplace?

Reply:
not the generated trimming test projects. Same goes for generated packaging tests.

Reply:
I meant test project in general don't binplace so why do you need to override the target here?

Reply:
I see, since the individual test projects are now importing the test.mobile.targets, one of t hose which we call to build the app bundle depends on GetBinPlaceTargetFramework, which is why if I don't define it I would get an error on those individual projec builds

Reply:
I can only see a dependency to `GetBinPlaceTargetFramework` in illink.targets which aren't imported?